frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -25,6 +25,19 @@
       data-qa-selector="add-existing-pane"
     >
     </op-add-existing-pane>
+
+    <button
```

Comment:
This should be a `type="button"`

Reply:
Using `<input type="button" />` is not possible since that does not allow nested dom which we have here. What would be the reason to prefer `type="button"`?

Reply:
What Ben means is `<button type="button">` as that carries the semantics for accessibility and allows activating it with both enter/space

Reply:
Understood now, thanks. Done in [d71cdcb](https://github.com/opf/openproject/pull/11601/commits/d71cdcbd7f0c2ee604c2ecaee1e6233e4b882aed)